tests/phpunit/tests/db/charset.php

```diff
@@ -677,21 +677,98 @@ public function test_check_ascii_false() {
 	/**
 	 * @ticket 21212
 	 *
+	 *
+	 * @dataProvider data_strip_invalid_text_for_column
+	 *
 	 * @covers wpdb::strip_invalid_text_for_column
 	 */
-	public function test_strip_invalid_text_for_column() {
```

Comment:
fix_problem_resolve_and_evolved

Reply:
Considering this to be a spam comment, hence resolving it.